qa/mixed-cluster/build.gradle

```diff
@@ -17,58 +17,88 @@
  * under the License.
  */
 
-import org.elasticsearch.gradle.test.RestIntegTestTask
 import org.elasticsearch.gradle.Version
+import org.elasticsearch.gradle.testclusters.RestTestRunnerTask
 
+apply plugin: 'elasticsearch.testclusters'
 apply plugin: 'elasticsearch.standalone-test'
 
-// This is a top level task which we will add dependencies to below.
-// It is a single task that can be used to backcompat tests against all versions.
-task bwcTest {
+tasks.register("bwcTest") {
   description = 'Runs backwards compatibility tests.'
   group = 'verification'
 }
 
-for (Version version : bwcVersions.wireCompatible) {
-  String baseName = "v${version}"
+configurations {
```

Comment:
Is there a reason we can't continue to use `RestIntegTestTask` which sets up this configuration and dependency for us?

Reply:
Mostly because it also creates a cluster and we want to set that up differently and explicitly here. 
I plan at looking at this at some point. 
Ideally we wouldn't need this and it could be a regular dependency, but we have to make the test runner read stuff off the classpath for that.